idf_component.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
   idf:
     version: ">=4.4"
   espressif/esp-nn:
-    version: "^1.1.0"
+    version: "^1.2.0"
```

Comment:
Please change this to `>= 1.1.1` as the change is added with version v1.1.1: https://components.espressif.com/components/espressif/esp-nn/versions/1.1.1